Sources/Starscream/WebSocket.swift

```diff
@@ -279,8 +279,9 @@ open class FoundationStream : NSObject, WSStream, StreamDelegate  {
             var peerNameLen: Int = 0
             SSLGetPeerDomainNameLength(sslContextOut, &peerNameLen)
             var peerName = Data(count: peerNameLen)
-            let _ = peerName.withUnsafeMutableBytes { (peerNamePtr: UnsafeMutablePointer<Int8>) in
-                SSLGetPeerDomainName(sslContextOut, peerNamePtr, &peerNameLen)
+            let _ = peerName.withUnsafeMutableBytes { (peerNamePtr:UnsafeMutableRawBufferPointer) in
```

Comment:
I don't want to nit pick but could we add space after `bytes:`

Reply:
added a space